editor/js/Sidebar.Properties.js

```diff
@@ -18,6 +18,54 @@ function SidebarProperties( editor ) {
 	container.addTab( 'scriptTab', strings.getKey( 'sidebar/properties/script' ), new SidebarScript( editor ) );
 	container.select( 'objectTab' );
```

Comment:
Oh! We probably do not need this line anymore?

Reply:
its needed for initializing, otherwise the default tabbedpanel value is the last option, in our case the SCRIPT tab

try:

1. Examples>Arkanoid
2. Select 'Ground' in outliner
3. You should see active tab in properties is SCRIPT, not OBJECT